assets/search/main.ejs

```diff
@@ -1,11 +1,10 @@
 <div class="container"
      ng-controller="SearchController as search"
-     ng-init="search.input.search = '<%- searchType %>'">
```

Comment:
Nah, don't remove this. I've fixed the bug that was stopping this from working, I think. Just need to test it.